src/translations/pt/core.js

```diff
@@ -0,0 +1,30 @@
+export default {
+  heroTitle: "Construa sistemas de design melhores",
```

Comment:
It is not necessary to translate "Design Systems" into Portuguese, as far as I know the expression in English is totally acceptable.

Reply:
Yes, but at the same time it's more accessible use terms in Portuguese. At components, for example, I used Portuguese terms with a tip in English.

Reply:
Sure, accessibility is a great point. I've just never heard the translated "Sistema de Design" being used in a conversation with (brazilian) programmers and designers. But the English tip is a cool idea too.